pubsub/consumer.go

```diff
@@ -8,33 +8,32 @@ import (
 	"time"
 
 	"github.com/ethereum/go-ethereum/log"
-	"github.com/go-redis/redis/v8"
 	"github.com/google/uuid"
 	"github.com/offchainlabs/nitro/util/stopwaiter"
+	"github.com/redis/go-redis/v9"
 	"github.com/spf13/pflag"
 )
 
 type ConsumerConfig struct {
 	// Timeout of result entry in Redis.
 	ResponseEntryTimeout time.Duration `koanf:"response-entry-timeout"`
-	// Duration after which consumer is considered to be dead if heartbeat
-	// is not updated.
-	KeepAliveTimeout time.Duration `koanf:"keepalive-timeout"`
+	// Minimum idle time after which messages will be autoclaimed
+	IdletimeToAutoclaim time.Duration `koanf:"Idletime-to-autoclaim"`
```

Comment:
Don't most koanf flag names use lowercase characters?

Reply:
my bad, will fix it